qtop.py

```diff
@@ -339,31 +357,34 @@ def fill_cpucore_columns(state_np_corejob, cpu_core_dict, id_of_username, max_np
         these positions are filled with '#'s.
         '''
         for core in own_np_empty_range:
-            cpu_core_dict['Cpu' + str(core) + 'line'] += '_'
+            core_user_map['Core' + str(core) + 'line'] += ['_']
         for core in non_existent_cores:
-            cpu_core_dict['Cpu' + str(core) + 'line'] += '#'
-    return cpu_core_dict
+            core_user_map['Core' + str(core) + 'line'] += ['#']
```

Comment:
good, but probably make them tunable characters (in the future, we may wish to replace with weird Unicode symbols!)